sdk/keyvault/keyvault-common/src/tracingHelpers.ts

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { Span, CanonicalCode } from "@opentelemetry/api";
+
+import { OperationOptions } from "@azure/core-http";
+import { createSpanFunction } from "@azure/core-tracing";
+
+/**
+ * Returns a function that can be used for tracing options.
+ *
+ * @param packagePrefix - The package prefix
+ *
+ * @internal
+ */
+export function trace(packagePrefix: string) {
```

Comment:
Perhaps `createTraceFunction` would match the naming convention of `createSpanFunction`.

Reply:
Yes, great idea! Will push that up shortly